packages/formula-editor/src/formula-editor.ts

```
@@ -0,0 +1,205 @@
+    );
+  }
+
+  formatFormula() {
+    if (!Boolean(this.content)) return;
+
+    this.content = this._parser.addParentheses(this.content) ?? this.content;
+    this.parseInput();
+    this.recommendations = [];
+  }
+
+  handleFocus(focus: Boolean) {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)